readme.md

```diff
@@ -246,73 +238,50 @@ If you complete 80% of the homework assigned over the duration of the class, and
+### Homework
 
-### Attendance
+There will be roughly 2 hours of homework every night. You'll be submitting the homework through Github, which we'll cover later on this week. All assignments will be due on Thursday at 9am. Your squad instructor will provide meaningful feedback on one of these assignments and also on your lab. Robin will check all the other homework and provide feedback and answer questions if you ask.
 
-You'll be taking your own attendance every morning. We have a system set up on Garnet that tracks the IP address from which you take attendance -- so we can tell if you're not actually in a classroom!
+It's **really, really** important that you submit homework even if -- especially if -- you didn't complete it. We would much rather you spend 2 hours taking a stab at something and turn it in unfinished than you spend 10 hours and turn it in perfect.
 
-Metro delays can happen. The solution is to leave every day earlier than you think you should. The instructors are on-time every day and we live all over the area. If we can do it, so can you!
+This is because without seeing your homework, we have no idea how you're progressing with the material, and so we can't get you the help you may need. Also if you don't turn in anything then it suggests to us that you're not making WDI a priority, and as such we shouldn't make you a priority.
 
-If you do arrive late, we'll ask that you not enter the classrooms until they go on break. Having folks coming in late is a huge distraction to the instructors and students, and if you come in late but can still sneak in then it's kind of a slap in the face to the students who are here on-time.
+Assignments are graded on whether or not you seem to have made a meaningful effort to do them. There are three possible grades for each assignment:
+  - Complete: meaningful effort was made. How "good" your code is doesn't have any impact on this.
+  - Incomplete: something was turned in, but meaningful effort was not made. No changes were made to the assignment, and/or it was blatantly plagiarized.
+  - Missing: nothing was turned in.
 
-### Instructor Communication
+### Attendance
 
-- Additional Instructor Support is available during Evening Office Hours from 6:00 - 8:00PM on Tuesday and Wednesday Nights
-- Instructors who are not scheduled for Evening Office Hours **may not** respond to code questions via Slack or in person, even if we're in the office.
-  - **Question**: Why would we do this?
-  - **Answer**: Work/Life Balance.
-- Feel free to send any messages to us and we'll reach out to you the next day.
-- Should you have a logistical question, you may contact Adrian Maseda
-  - He is the **Student Communication Manager** for your cohort
+You'll be taking your own attendance every morning. We have a system set up on Garnet that tracks the IP address from which you take attendance -- so we can tell if you're not actually in a classroom!
```

Comment:
feels a little big brothery - maybe make a joke about monitoring IP addresses without explicityl saying we are tracking you